datafusion/core/src/physical_optimizer/enforce_distribution.rs

```diff
@@ -1492,9 +1491,7 @@ pub(crate) mod tests {
             if self.expr.is_empty() {
                 vec![None]
             } else {
-                vec![Some(PhysicalSortRequirement::from_sort_exprs(
-                    self.expr.iter(),
-                ))]
+                vec![Some(LexRequirement::from(self.expr.clone()))]
```

Comment:
This is a pretty good example of the potential improvement -- it is now clear this function is returning a `LexRequirement` and that it clones the underlying `LexOrder` structure